src/encoded/schemas/experiment_seq.json

```diff
@@ -122,6 +122,13 @@
             "type": "number",
             "lookup": 113,
             "use_for": ["MARGI"]
+        },
+        "rna_strandness":{
```

Comment:
Should be 'strandedness'
suggest that remove the rna_ from the propname and just go with 'strandedness'